pkg/controller/migplan/pvlist.go

```diff
@@ -376,7 +377,7 @@ func (r *ReconcileMigPlan) getClaims(client compat.Client, plan *migapi.MigPlan)
 	}
 
 	alreadyMigrated := func(pvc core.PersistentVolumeClaim) bool {
-		if planuid, exists := pvc.Labels[migapi.MigMigrationLabel]; exists {
+		if planuid, exists := pvc.Labels[migapi.MigPlanLabel]; exists {
```

Comment:
Why the switch to the migplan label?

Reply:
Because the MigMigrationLabel didn't make sense. Once you migrate, you cannot migrate again, unless you rollback. At which point the migration label is overwritten to the new MigMigration. To me it made much more sense to mark the PVCs with the MigPlan since that will be constant regardless of how many times you migrate and roll back.